app/src/reducers/layers.js

```diff
@@ -100,7 +101,7 @@ export default function (state = initialState, action) {
       });
 
       if (newState.layerPanelEditMode === false) {
-        newState.workspaceLayers = _.cloneDeep(state.workspaceLayers);
+        newState.workspaceLayers = cloneDeep(state.workspaceLayers);
 
 
         newState.workspaceLayers.forEach((layer) => {
```

Comment:
same as above. Perhaps here, we could use `map` instead of `forEach` and return new objects where needed,